firmware/src/dimmer_model.c

```diff
@@ -1315,7 +1315,7 @@ static const PIDDescriptor SUBDEVICE_PID_DESCRIPTORS[] = {
   {PID_CLEAR_STATUS_ID, (PIDCommandHandler) NULL, 0u,
     DimmerModel_ClearStatusId},
   {PID_SUB_DEVICE_STATUS_REPORT_THRESHOLD,
-    DimmerModel_GetSubDeviceReportingThreshold, 1u,
```

Comment:
Ah of course, because this would flag the normal request as incorrect. I was confused as you said you weren't doing WithData type requests before.